lib/rules/role-img-alt.json

```diff
@@ -0,0 +1,18 @@
+{
+	"id": "role-img-alt",
+	"selector": "[role='img']:not(svg):not(img):not(area):not(input):not(object)",
+	"tags": [
+		"cat.text-alternatives",
+		"wcag2a",
+		"wcag111",
+		"section508",
+		"section508.22.a"
+	],
+	"metadata": {
+		"description": "Ensures [role='img'] elements have alternate text",
+		"help": "Images must have alternate text"
```

Comment:
Please change this to `[role='img'] elements have an alternative text`, to distinguish it from other "image" rules.